app/assets/stylesheets/govuk_publishing_components/components/helpers/_variables.scss

```diff
@@ -25,3 +25,5 @@ $gem-focus-colour: $govuk-focus-colour;
 
 $gem-error-colour: govuk-colour("red");
 $gem-success-colour: govuk-colour("green");
+
+$gem-hover-dark-background: #dddcdb;
```

Comment:
If this is only used in the action link component right now I'd put it at the top of the action link sass - no need to make it more visible until we're using it elsewhere.

Reply:
Makes sense, moved and updated